design-system-docs/Gemfile

```diff
@@ -13,9 +13,6 @@ gem "puma"
 
 gem "citizens_advice_components", path: "../engine"
 
-# https://github.com/citizensadvice/design-system/issues/3145
-gem "view_component", "3.8.0"
```

Comment:
Should the PR include a note for why you've removed the `view_component` dependency? Is `citizens_advice_components` now using 3.9.0+ internally?

Reply:
It will use the latest version of view component (we technically support both 2.x and 3.x) but/and yep let me add a note in the commit and PR description.